content/apidocs-mxsdk/apidocs/team-server-api.md

```diff
@@ -4,203 +4,411 @@ category: "API Documentation"
 menu_order: 65
 ---
 
+{{% alert type="warning" %}}
+The API description on this page refers to the new team server API implementation. You can access the documentation for the previous team server API implementation [here](old-version/team-server-api.md).
+{{% /alert %}}
+
 ## 1 Introduction
 
-The team server API allows you to retrieve the information (branches, revisions) of application models stored in our team server. You always access an application model via the context of an application (see the Deploy API for more information about retrieving applications and application identifiers).
-The image below provides a domain model representation of the concepts discussed below and how these are related:
-![](attachments/131076/425989.png)
+The team server API allows you to retrieve the information (branches, commits) of application models stored in our team server.
 
-{{% alert type="warning" %}}
-The team server API is only available to *licensed* apps which are running in a Mendix Cloud.
-{{% /alert %}}
+The base URL for all Team Server API endpoints is:
+```
+https://teamserver.api.mendix.com/v1
+```
+
+All available endpoints are:
+| Method | Endpoint | Description |
+| --- | --- | --- |
+| GET | [`/repositories/<projectId>`](#41-retrieve-repository-info) | Get repository information for a project |
```

Comment:
Need to add anchors {#XXX} manually to the linked sections, and use anchors for cross-references.

Reply:
Added the missing manual anchors